herddb-core/src/main/java/herddb/server/Server.java

```diff
@@ -338,7 +338,7 @@ protected CommitLogManager buildCommitLogManager() {
 
         switch (mode) {
             case ServerConfiguration.PROPERTY_MODE_LOCAL:
-                return new MemoryCommitLogManager();
+                return new MemoryCommitLogManager(false);
```

Comment:
this saves the dummy serialization to the MemoryCommitLog: dummy serialization is useful for own tests (and still is in place in tests) but it is useless for our users